src/chrome/content/rules/Craigslist.org.xml

```diff
@@ -1,54 +1,38 @@
 <!--
 	Other Craigslist rulesets:
-
 		- Craigslist.ca.xml
 		- Craigslist.co.uk.xml
 
+	(May 2016) Craigslist only supports https on its .org, .ca, and .co.uk
+	domains. geo.craigslist.org redirects to regional sites by sending protocol-
+	relative redirects such as "//toronto.craigslist.ca". Unfortunately, it does this
+	even when the destination doesn't support https, such as helsinki.craigslist.fi,
+	making a downgrade to http couse travis error:
+	Too many redirects while fetching 'http://geo.craigslist.org/'
+	So exclusion only.
```

Comment:
I'm not sure if this is still true, this needs further testing.
`toronto.craigslist.ca` WFM on HTTPS and also `https://geo.craigslist.org/iso/fi` redirects correctly.